src/features/projectsV2/ProjectsMap/index.tsx

```diff
@@ -205,37 +212,58 @@ function ProjectsMap(props: ProjectsMapProps) {
     },
     [plantLocations, props.page, selectedPlantLocation]
   );
-
+  /**
+   * This onClick handler is responsible for:
+   * - Selecting: point plant locations(single tree), polygon plant locations(multi tree), or project sites
+   * - Deselecting: point plant locations ,sample point plant locations, other interventions(point geometry)
+   */
   const onClick = useCallback(
```

Comment:
This comment is helpful, but it would be useful to know when this click handler will be invoked. Can you update the comment?